src/core/ext/transport/binder/wire_format/binder_android.h

```diff
@@ -127,6 +123,6 @@ class TransactionReceiverAndroid final : public TransactionReceiver {
 
 }  // namespace grpc_binder
 
-#endif  // defined(ANDROID) || defined(__ANDROID__)
+#endif
```

Comment:
`#endif  // GRPC_SUPPORT_BINDER_TRANSPORT` for consistency

Reply:
Cannot add that comment because include guard checker will think it is include guard and rename it for us

Reply:
Ack.